### ISSUE TITLE
Physics Model / Grid Documentation - Update/Complete?  

### ISSUE THREAD

**Issue by LanaEid** [BEAST-Fitting/beast#505]
Can the documentation for https://beast.readthedocs.io/en/latest/beast_grid_format.html be updated/completed?  There are a lot of question marks and placeholders, and generally incomplete information.  _(For reference, I [am new to using the BEAST and] am trying to find everything I need to simulate photometric observations.  As far as I can tell, I need to use "physics model / grid," "observation model," and then "gen_SimObs_from_sedgrid" (each one seems to need the one(s) before it)._
Thanks!

**Comment by karllark** [comment]
You are correct that the docs do need updating.  The question marks are mainly due to not being sure about the units.  If you have specific questions, please ask them here.  

For simulating observations, you are correct in what is needed.
